client/src/RatingsAndReviews/StaticStarList.jsx

```diff
@@ -8,11 +8,11 @@ export default function Stars({ productId, ratingInt, returnAvgRating }) {
   useEffect(() => {
     if (productId) {
       axios.get(`/reviews?product_id=${productId}&count=9999`)
-        .then((results) => {
-          const ratingSum = results.data.results.reduce((accum, val) => (
+        ?.then((results) => {
```

Comment:
Good use of inline '?' conditional operator to check for valid return before .then